jobs/generate-release-notes.Jenkinsfile

```diff
@@ -0,0 +1,70 @@
+@Library('github.com/coreos/coreos-ci-lib@master') _
+
+config_repo = "coreos/fedora-coreos-config"
+releng_script_repo = "coreos/fedora-coreos-releng-automation"
+branches = [
+    "testing-devel"
+    "next-devel"
+]
+botCreds = "github-coreosbot-token"
+
+cosaPod {
+    // set up fedora-coreos-releng-automation repository
+    shwrap("""
+    mkdir fcos-releng
+    git clone --branch master https://github.com/${releng_script_repo} fcos-releng"
+    git config --global user.name "CoreOS Bot"
+    git config --global user.email "coreosbot@fedoraproject.org"
+    """)
+
+    parallel branches.collectEntries { branch -> [branch, {
+        // build 'release-notes.yaml' using 'release-notes.d/' under testing-devel and next-devel
+        stage("Build 'release-notes.yaml'") {
+            shwrap("mkdir ${branch}")
+
+            dir(branch) {
+                shwrap("cosa init --branch ${branch} https://github.com/${config_repo}")
```

Comment:
Do we need to use `cosa` here? Should we just use `git clone` instead?

Reply:
All I needed was a FCOS config repo so no hard need for a `cosa init`.

Reply:
Yeah, let's just use `git clone --depth=1` to keep things simple then.